### ISSUE TITLE
Update VAMC Police template with components

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#16958]
## Description
VAMC VA Police pages need the following updates to components:

Example pages:

- [Boston Police](https://www.va.gov/boston-health-care/va-police/)
- [Minneapolis Police](https://www.va.gov/minneapolis-health-care/va-police/)

1. Breadcrumbs (I think this might not be possible? I just am very surprised I missed this first round)
   - These are currently not using a component at all but they should be using [the v3 component](https://design.va.gov/components/breadcrumbs#examples---v3) 
2. Link
   - There are instances of links within accordions that are currently not using a component at all. These should be using [the v1 component](https://design.va.gov/components/link/) 
3. Accordion (I wasn't aware there was a v3 when we first did this which is how this slipped by)
   - The accordion section is using the v1 component but should be updated [to v3](https://design.va.gov/components/accordion#examples---v3). NOTE: This will automatically happen on 2/16 if not manually done prior. 

## Acceptance Criteria
- [ ] v3 Breadcrumb component has been added, if there are not technical limitations
- [ ] Link component has been added
- [ ] Accordion component has been updated
- [ ] Requires design review
- [ ] Requires accessibility review

## Team
Please check the team(s) that will do this work.

- [ ] CMS Team
- [ ] Public Websites
- [x] Facilities
- [ ] Accelerated Publishing

**Comment by laflannery** [comment]
@xiongjaneg Some components on the Plice page slipped by me during mmvp. Since we aer currently iterating on this, it might be efficient to pull this in during mvp?

**Comment by xiongjaneg** [comment]
@laflannery Yes please, there's the [Police MVP epic](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/14327) and any tickets can go there and I'll get them in refinement and beyond. Thank you!

**Comment by maxx1128** [comment]
I estimate this ticket at a **2** for moving a few relatively simple elements over to components and upgrading the accordions.

**Comment by laflannery** [comment]
Accordions have already been updated to v3, I believe from [this PW ticket](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/16980)
![Screenshot 2024-03-27 at 11 18 53 AM](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/106678594/c1de1c33-ee9c-4782-820f-904ac829cfd0)

**Comment by laflannery** [comment]
Also removed the link component from this ticket, after working with PW, the links I noted are CMS content and we are limited with what we can do with that type of content

Because of these 2 updates, there is now nothing left to do on this ticket, closing
